applications/serial_lte_modem/doc/slm_description.rst

```diff
@@ -349,12 +364,12 @@ Connecting with an external MCU
 
    This section does not apply to Thingy:91 as it does not have UART2.
 
-If you run your user application on an external MCU (for example, an nRF52 Series development kit), you can control the modem on nRF9160 directly from the application.
+If you run your user application on an external MCU (for example, an nRF52 Series development kit), you can control the modem on an nRF91 Series device directly from the application.
 See the :ref:`slm_shell_sample` for a sample implementation of such an application.
 
 To connect with an external MCU using UART_2, change the configuration files for the default board as follows:
 
-* In the :file:`nrf9160dk_nrf9160_ns.conf` file::
+* In the :file:`nrf9160dk_nrf9160_ns.conf` or :file:`nrf9161dk_nrf9161_ns.conf`, depending on the DK you are using::
 
      # Use UART_0 (when working with PC terminal)
```

Comment:
This could likely be in format:

"To connect with an external MCU using UART_2, change the configuration files, depending on the DK you are using:"

In the :file:`nrf9160dk_nrf9160_ns.conf` file::

# Use UART_0 (when working with PC terminal)
# unmask the following config
#CONFIG_UART_0_NRF_HW_ASYNC_TIMER=2
#CONFIG_UART_0_NRF_HW_ASYNC=y
**#CONFIG_SLM_WAKEUP_PIN=6
#CONFIG_SLM_INDICATE_PIN=2**

# Use UART_2 (when working with external MCU)
# unmask the following config
CONFIG_UART_2_NRF_HW_ASYNC_TIMER=2
CONFIG_UART_2_NRF_HW_ASYNC=y
CONFIG_SLM_WAKEUP_PIN=31
CONFIG_SLM_INDICATE_PIN=30

In the :file:`nrf9161dk_nrf9161_ns.conf` file::

# Use UART_0 (when working with PC terminal)
# unmask the following config
#CONFIG_UART_0_NRF_HW_ASYNC_TIMER=2
#CONFIG_UART_0_NRF_HW_ASYNC=y
**#CONFIG_SLM_WAKEUP_PIN=8
#CONFIG_SLM_INDICATE_PIN=0**

# Use UART_2 (when working with external MCU)
# unmask the following config
CONFIG_UART_2_NRF_HW_ASYNC_TIMER=2
CONFIG_UART_2_NRF_HW_ASYNC=y
CONFIG_SLM_WAKEUP_PIN=31
CONFIG_SLM_INDICATE_PIN=30

Reply:
updated. Please check.